src/sql/workbench/contrib/notebook/browser/calloutDialog/linkCalloutDialog.ts

```diff
@@ -45,6 +45,7 @@ export class LinkCalloutDialog extends Modal {
 		dialogPosition: DialogPosition,
 		dialogProperties: IDialogProperties,
 		private readonly _defaultLabel: string = '',
+		private readonly _defaultLinkLabel: string = '',
```

Comment:
The link isn't really a label - I'd suggest `_defaultLinkUrl` to match the name of the input